packages/@aws-cdk/assets/package.json

```diff
@@ -56,4 +60,4 @@
     "@aws-cdk/cx-api": "^0.8.2"
   },
   "homepage": "https://github.com/awslabs/aws-cdk"
-}
+}
```

Comment:
Your editor has stripped new line at end of file on a few files making the diff larger than needed.

Reply:
✔️